app/src/test/java/com/appunite/loudius/ui/reviewers/ReviewersViewModelTest.kt

```diff
@@ -91,104 +98,160 @@ class ReviewersViewModelTest {
         fun `GIVEN correct initial values WHEN init THEN pull request number is correct`() {
             viewModel = createViewModel()
 
-            assertEquals("correctPullRequestNumber", viewModel.state.pullRequestNumber)
+            expectThat(viewModel.state)
+                .get(ReviewersState::pullRequestNumber)
+                .isEqualTo("correctPullRequestNumber")
         }
 
         @Test
         fun `GIVEN correct initial values WHEN init starts THEN state is loading`() = runTest {
-            coEvery { repository.getReviews(any(), any(), any()) } coAnswers { neverCompletingSuspension() }
+            coEvery {
+                repository.getReviews(
+                    any(),
+                    any(),
+                    any(),
+                )
+            } coAnswers { neverCompletingSuspension() }
             viewModel = createViewModel()
 
-            assertEquals(true, viewModel.state.isLoading)
-            assertEquals(emptyList<Reviewer>(), viewModel.state.reviewers)
+            expectThat(viewModel.state) {
+                get(ReviewersState::isLoading).isTrue()
+                get(ReviewersState::isError).isFalse()
+                get(ReviewersState::reviewers).isEmpty()
+            }
         }
 
         @Test
         fun `GIVEN no reviewers WHEN init THEN state is correct with no reviewers`() {
-            coEvery { repository.getReviews(any(), any(), any()) } returns Result.success(emptyList())
-            coEvery { repository.getRequestedReviewers(any(), any(), any()) } returns Result.success(RequestedReviewersResponse(emptyList()))
+            coEvery {
+                repository.getReviews(
+                    any(),
+                    any(),
+                    any(),
+                )
+            } returns Result.success(emptyList())
+            coEvery {
+                repository.getRequestedReviewers(
+                    any(),
+                    any(),
+                    any(),
+                )
+            } returns Result.success(RequestedReviewersResponse(emptyList()))
```

Comment:
**Question:** Was it auto-formatting or do you prefer such a format? Cause it's a lot of additional lines.